templates/login.liquid

```diff
@@ -0,0 +1,174 @@
+        visibility: hidden;
+      }
+      @media (min-width: 1070px) {
+        .forum-instructions-balancer {
+          width: calc(var(--forum-instructions-width) + var(--forum-instructions-margin));
+        }
+      }
+      .contact-us {
+        width: 100%;
+        height: calc(2em+5px);
+        background-color: #dae0e2;
+        padding: 10px;
+        border-radius: 5px;
+        text-align: center;
+        position: relative;
+        z-index: 5;
+      }
+      @media (min-width: 480px) {
+        .contact-us {
+          margin-top: 20px;
+          margin-bottom: 20px;
+          width: 400px;
+        }
+      }
+      @media (min-width: 1070px) {
+        .contact-us {
+          margin-bottom: 0;
+        }
+      }
+    </style>
+    <script>
+      function hideForumDetails() {
+        const details = document.getElementById("forum-instructions-details");
+        details.style.display = "none";
+        const showMore = document.getElementById("forum-instructions-show-more");
+        showMore.style.display = "block";
+      }
+      function showForumDetails() {
+        const details = document.getElementById("forum-instructions-details");
+        details.style.display = "block";
+        const showMore = document.getElementById("forum-instructions-show-more");
+        showMore.style.display = "none";
+      }
+    </script>
+  </head>
+  <body>
+    <div class="ea-layout {% if application.name contains "Forum" %}ea-layout-forum{% else %}ea-layout-contact{% endif %}">
+      {% if application.name contains "Forum" %}
+        <div class="forum-instructions">
+          <p>Welcome to our new login page!</p>
+          <div class="forum-instructions-details" id="forum-instructions-details">
+            <div class="forum-instructions-details-text">
+              <p>
+                You <a href="https://forum.effectivealtruism.org/posts/AhzgWBc9hDF8grnGf/forum-update-new-features-june-2021#Unified_logins_for_effectivealtruism_org">
+                now log in</a> to the EA Forum using an EffectiveAltruism.org account,
+                which means you can use the same account for the Forum, EA Funds,
+                Giving What We Can, and EA Global applications.
+              </p>
+              <p>
+                It's no longer possible to login with your username. Please use your
+                email address instead. If you can't remember the email address you
+                signed up with, or you have any other issues, please get in touch with
+                <a href="https://forum.effectivealtruism.org/contact">support</a>.
+              </p>
+            </div>
+            <div class="forum-instructions-hide-this">
+              <strong><a onclick="hideForumDetails()">Hide this</a></strong>
+            </div>
+          </div>
+          <div class="forum-instructions-show-more" id="forum-instructions-show-more">
+            <strong><a onclick="showForumDetails()">Show more</a></strong>
```

Comment:
Maybe 'Login help'? or something. Otherwise if the element isn't shown (e.g. someone misclicks before reading the page) it's not clear what you're showing.

Reply:
Oh wait, I see, the text 'Welcome to our new login page!' is shown at all times. Scratch that.